pkgs/top-level/perl-packages.nix

```diff
@@ -7304,6 +7304,20 @@ let
     };
   };
 
+  DevelOverrideGlobalRequire = buildPerlPackage {
+    pname = "Devel-OverrideGlobalRequire";
+    version = "0.001";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/D/DA/DAGOLDEN/Devel-OverrideGlobalRequire-0.001.tar.gz";
+      sha256 = "0791892de3ae292af4a94e382f21db1ee88210875031851e6ea82c3410785ef9";
+    };
+    meta = {
+      homepage = "https://metacpan.org/release/Devel-OverrideGlobalRequire";
```

Comment:
This redirects to https://metacpan.org/dist/Devel-OverrideGlobalRequire. I'm not familiar with CPAN so not sure which is the one to use, just a heads up

Reply:
/release is what `nix-generate-from-cpan` generates, and it seems like almost all the other links in `perl-packages.nix` also use the /release path, so I think /release is fine to leave